### DISCUSSION TITLE
Dashboard: add chart for unique users (MAU/WAU/DAU)

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#3472] [Ideas]
### Describe the feature or potential improvement

It is useful to track how many unique users have used the application according to the traces logged to a project in Langfuse.

Background on users: https://langfuse.com/docs/tracing-features/users

**Comment by ThomasBTR** [comment]
We would be really happy to have this feature !

**Comment by 2underscores** [comment]
+1 would be great if widget builder supported this

**Comment by Beg1nner4ever** [comment]
It would really be great, it's one of the most useful KPIs to see how the audience of our project is evolving

**Comment by vldF** [comment]
+1, implement this one please
